epilogue-runtime/src/main/java/edu/wpi/first/epilogue/EpilogueConfiguration.java

```diff
@@ -23,6 +26,12 @@ public class EpilogueConfiguration {
    */
   public DataLogger dataLogger = new NTDataLogger(NetworkTableInstance.getDefault());
 
+  /** The period Epilogue will log at. */
+  @SuppressFBWarnings public Measure<Time> loggingPeriod;
+
+  /** The offset from the periodic run that Epilogue will log at. */
+  @SuppressFBWarnings public Measure<Time> loggingPeriodOffset;
```

Comment:
Referencing these in a test or examples would remove the need to suppress spotbugs warnings.

The javadoc for these fields should also detail the default behavior (ie how epilogue behaves when they're null) and how it interacts with `bind`